tests/urllib_utf8_test.py

```diff
@@ -38,6 +39,19 @@ def assert_params_equal(left, right):
                         urllib.urlencode(utf8_query))
 
 
+def test_urlencode_lists():
+    query = {'exprs': ['foo:123', u'bär:222']}
```

Comment:
Please add a non-latin1 character (i think i use monkeyface elsewhere), and add a UTF-8 (ie byte-literal, `b''`) entry.
Often bugs crop up with unicode vs utf-8 inputs.